addons/cert-manager/oidc-proxy-certificate/README.md

```diff
@@ -0,0 +1,20 @@
+# Cert Manager for OIDC Proxy Certificate Add-on
+
+This add-on add support for Cert Manager Certificate resource for the OIDC Proxy certificate instead of
+relay on an autogenerated one.  
+To work properly it need both the Cert Manager and oidc-proxy module.
```

Comment:
```suggestion
This add-on adds support for Cert Manager Certificate resource for the OIDC Proxy certificate instead of
relying on an autogenerated one.  
To work properly it needs both the Cert Manager and oidc-proxy module.
```